mkdocs.yml

```
@@ -18,6 +18,7 @@ nav:
     - Build customer loyalty programs: customer-360/loyalty-rewards/index.md
   - Cybersecurity:
     - Detect and analyze SSH attacks: cybersecurity/SSH-attack/index.md
+    - Detect Slowloris DDoS attacks: cybersecurity/ddos/index.md
```

Hide in nav until PMM provides copy & graphics

```suggestion
    #- Detect Slowloris DDoS attacks: cybersecurity/ddos/index.md
```